ui-app/client/src/components/util/properties.ts

```diff
@@ -326,6 +326,7 @@ const COMPONENT_STYLE_GROUPS: { [key: string]: Array<string> } = {
 		'letterSpacing',
 	],
 	background: [
+		'background',
```

Comment:
Remove background property. We don't want any shorthand properties in style properties.